WordPressKit/HTTPClient.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+
+public typealias WordPressAPIResult<R, E: LocalizedError> = Result<R, WordPressAPIError<E>>
```

Comment:
Nitpick. I've seen Swift generics use more verbose labels, e.g. [`Result<Success, Failure>`](https://developer.apple.com/documentation/swift/result)

```suggestion
public typealias WordPressAPIResult<Response, Error: LocalizedError> = Result<Response, WordPressAPIError<Error>>
```

Not sure if `Error` will work or get confused with `Swift.Error`. If it doesn't work, then maybe `Failure`?